File: src/main/java/br/com/pedidos/services/ModeloProdutoService.java
package br.com.pedidos.services;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import br.com.pedidos.model.ModeloProduto;
import br.com.pedidos.repository.ModeloProdutoRepository;

@Service
public class ModeloProdutoService {
	
	@Autowired
	ModeloProdutoRepository modeloProdutoRepository;
	
	public ModeloProduto salvarModelo(ModeloProduto modeloProduto) {
		modeloProdutoRepository.save(modeloProduto);
		
		return modeloProduto;
	}
	
	public List<ModeloProduto> getTodosModelos() {
		return this.modeloProdutoRepository.findAll();
	}

}

File: src/main/webapp/js/services/modeloProdutoService.js
app.factory("modeloProdutoService", function ($http) {

	var _salvarModelo = function (modelo) {

		return $http.post("/modeloProduto", modelo);
	};

	var _getModelos = function () {
		return $http.get("/modeloProduto");
	}

	var _getModeloPorId = function (id) {
		return $http.get("/modeloProduto/" + id);
	}



	return {
		salvarModelo : _salvarModelo,
		getModelos : _getModelos,
		getModeloPorId : _getModeloPorId
	};

}); 